neo/IO/Caching/FIFOSet.cs

```diff
@@ -46,11 +48,11 @@ public bool Contains(T item)
             return dictionary.Contains(item);
         }
 
-        public void ExceptWith(IEnumerable<UInt256> hashes)
```

Comment:
In fact, this was wrong, right? Aheuaheuaea
It should be a template `T`. 

Nice catch.